packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -371,7 +355,7 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
         bytes memory hookData =
             abi.encodeCall(IFastBridgeRecipient.fastBridgeTransferReceived, (token, amount, callParams));
         // This will bubble any revert messages from the hook function
-        bytes memory returnData = Address.functionCallWithValue({target: recipient, data: hookData, value: msgValue});
+        bytes memory returnData = Address.functionCallWithValue({target: recipient, data: hookData, value: msg.value});
         // Explicit revert if no return data at all
         if (returnData.length == 0) revert RecipientNoReturnValue();
```

Comment:
this would also confirm that the recipient must be a smart contract, not an EOA - correct?

Reply:
Correct! This is handled by the OZ library here & here:

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v5.0/contracts/utils/Address.sol#L83-L89

https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v5.0/contracts/utils/Address.sol#L114-L129